src/Hosting.Services.Web/Middlewares/ResponseHeadersMiddleware.cs

```diff
@@ -25,6 +26,12 @@ private Task SetResponseHeaders(object state)
 			HttpResponse response = (HttpResponse)state;
 			response.Headers.Add("X-Machine", m_context.MachineId);
 			response.Headers.Add("X-BuildVersion", m_context.BuildVersion); //Renamed from X-OfficeVersion
+
+			if (Activity.Current != null)
+			{
+				response.Headers.Add("X-TraceId", Activity.Current.TraceId.ToString());
```

Comment:
Isn't it available as a standard header?

Reply:
AT least not for our service

<img width="910" alt="image" src="https://user-images.githubusercontent.com/8594708/230470980-f3999ea5-1b75-4e2d-ad33-3f779c783b9b.png">

Reply:
I've found this from a quick search: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/getting-started-aspnetcore/README.md
Admittedly, the instrumentation is still pre-release, and looks like there are some performance concerns, but I'd rather go along with standardization effort than create a custom solution.